security-keycloak-authorization-quickstart/src/main/java/org/acme/security/keycloak/authorization/AdminResource.java

```diff
@@ -1,3 +1,18 @@
+/**
```

Comment:
@survivant I'm not sure the license headers have to be added to the main source files, if they have to then it is likelyly have to be `2019-2021` years, but please wait for @gsmet to confirm

Reply:
removed